File: fixtures/repositories/python-genuine-bug-fix/base/reproduce.py
import json
import os

from calculator import add

if add(2, 2) != 4:
    print("bug reproduced")
    print(json.dumps({"nonce": os.environ.get("PATCHPROOF_NONCE"), "status": "assertion_failed"}))
    raise SystemExit(1)

print(json.dumps({"nonce": os.environ.get("PATCHPROOF_NONCE"), "status": "assertion_passed"}))

File: fixtures/repositories/genuine-bug-fix/base/reproduce.js
import { add } from "./math.js";

if (add(2, 2) !== 4) {
  console.error("bug reproduced");
  console.log(JSON.stringify({ nonce: process.env.PATCHPROOF_NONCE, status: "assertion_failed" }));
  process.exit(1);
}

console.log(JSON.stringify({ nonce: process.env.PATCHPROOF_NONCE, status: "assertion_passed" }));
